atom/browser/ui/message_box_gtk.cc

```diff
@@ -154,13 +168,16 @@ class GtkMessageBox : public NativeWindowObserver {
   }
 
   CHROMEGTK_CALLBACK_1(GtkMessageBox, void, OnResponseDialog, int);
+  CHROMEGTK_CALLBACK_1(GtkMessageBox, void, OnCheckboxToggled, gpointer);
 
  private:
   atom::UnresponsiveSuppressor unresponsive_suppressor_;
 
   // The id to return when the dialog is closed without pressing buttons.
   int cancel_id_;
 
+  bool checkbox_checked_ = false;
```

Comment:
I think this should be initialized in the constructor like `checkbox_checked_(false)` after `cancel_id_` is assigned there for consistency with the rest of the codebase.